main.cpp

```diff
@@ -485,6 +486,21 @@ void ChannelStrip::acceptAudioInput(bool isConnected, bool isMono)
     std::cout << "How loud can it go? " << maxGainRange << "db" << std::endl;
 }
 
+float ChannelStrip::addChannel(int numChannels)
+{
+    for (int i = 0; i < numChannels; i++)
+    {
+        int channel = 0;
+
+        if (channel > 1)
```

Comment:
`channel` is never `> 1` because you're declaring it and initializing it to the value of `0` right above this line.
declare `channel` before the `for()` loop.
- [ ] revise

Reply:
Ok, not sure what I was thinking here. Changed it to channel <= 1 and initialized to 1 outside the loop. The function now adds a channel.